go/vumitools/account/migrations.py

```diff
@@ -109,3 +109,42 @@ def reverse_from_5(self, mdata):
         mdata.set_value('$VERSION', 4)
 
         return mdata
+
+    def migrate_from_5(self, mdata):
+        """
+        Add the account_types field and default it to `ACCOUNT_TYPE_NORMAL`
+        """
+
+        # Copy stuff that hasn't changed between versions
+        mdata.copy_values(
+            'username', 'created_at', 'msisdn', 'confirm_start_conversation',
+            'tags', 'event_handler_config', 'routing_table',
+            'can_manage_optouts', 'disable_optouts')
+        mdata.copy_indexes('tagpools_bin', 'applications_bin')
+
+        # import here to avoid circular import
+        from go.vumitools.account.models import UserAccount
```

Comment:
There is potential for this to break if ``UserAccount.ACCOUNT_TYPE_NORMAL`` goes away or changes at some point, but I think that's okay since there are tests.